Cabal/Distribution/Simple/GHC/ImplInfo.hs

```diff
@@ -65,20 +66,26 @@ ghcVersionImplInfo ver = GhcImplInfo
   , flagProfAuto         = v >= [7,4]
   , flagPackageConf      = v <  [7,5]
   , flagDebugInfo        = v >= [7,10]
+  , supportsPkgEnvFiles  = v >= [8,0,2] -- broken in 8.0.1, fixed in 8.0.2
```

Comment:
Note: to test with current GHC 8.0.x branch you need to edit this to be

    v > [8,0,1]

indeed, perhaps it should just be that anyway.